lib/ui/components/current_timer/CurrentTimerDisplay.svelte

```diff
@@ -20,7 +22,13 @@
     {#if timer}
       <div id="description">
         {#if timer.description}
-          <span class="timer-description">{timer.description}</span>
+          <span class="timer-description">
+            {#if $settingsStore.parseMarkdown}
+              {@html renderMarkdown(timer.description)}
```

Comment:
This correctly renders the Markdown as a link, but clicking the link doesn't actually open the related page.

Reply:
That's odd. I'll investigate.